frappe/patches/v13_0/remove_twilio_settings.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+
+import frappe
+
+
+def twilio_settings_doctype_in_integrations() -> bool:
+	"""Check Twilio Settings doctype exists in integrations module or not.
+	"""
+	return frappe.db.exists("DocType", {'name': 'Twilio Settings', 'module': 'Integrations'})
+
+def execute():
+	"""Add missing Twilio patch.
+
+	While making Twilio as a standaone app, we missed to delete Twilio records from DB through migration. Adding the missing patch.
+	"""
+	frappe.delete_doc_if_exists('DocType', 'Twilio Number Group')
+	if twilio_settings_doctype_in_integrations():
+		frappe.delete_doc_if_exists('DocType', 'Twilio Settings')
+		frappe.db.sql("delete from `tabSingles` where `doctype`=%s", 'Twilio Settings')
```

Comment:
@surajshetty3416 , I was expecting `Twilio Settings` doctype  records from `tabSingles` will be removed when we delete that doctype. but, that is not happening somehow. Is that the issue that need to be fixed or is that the behaviour of single DocTypes?

Reply:
`delete_doc_if_exists ` should have deleted that single doctype from `tabSingles` table since it internally calls [delete_from_table](https://github.com/frappe/frappe/blob/develop/frappe/model/delete_doc.py#L160)

Reply:
Okay, then that must be a bug. will take care of that!